### PR TITLE
[IMPROVEMENT] Modify install.sh to read input variables and log related warnings if variable not set

### PR DESCRIPTION
**Pull request by Tarun-Arora**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---

This PR includes the following changes:
- Create a function `userInput` to take inputs from the user, and based on the function arguments, we can mark the variables as required or not.
- In case the variables are marked not required and user doesn't set them, we log variable-specific warnings to the user.

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -49,6 +49,28 @@ do
         done
     fi
 done
+
+RED='\033[0;31m'
+NC='\033[0m'
```
Comment (thealphadollar) [inline install/install.sh:54]: Thanks for the color coding 💯

**Review by thealphadollar — COMMENTED** [review]
The changes look good to me. Can you please add a few screenshots of the different cases, just for the input entering (with the colour, error, loop, etc.):

- Error
- Warning
- Successful

**Comment by Tarun-Arora** [comment]
> The changes look good to me. Can you please add a few screenshots of the different cases, just for the input entering (with the colour, error, loop, etc.):
> 
> * Error
> * Warning
> * Successful

Yes Sure.
Screenshot_20221007_215700

File: install/install.sh [hunk 2]
```diff
@@ -60,33 +82,32 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -s -e -r -p  "Password of the 'root' user of MySQL: " -i "" db_root_password
+read -s -e -r -p  "$(tput bold)Password of the 'root' user of MySQL:$(tput sgr0) " -i "" db_root_password
 echo ""
 # Verify password
 
 supress_warning=$(mysql_config_editor set --login-path=root_login --host=localhost --user=root --password "${db_root_password}") >> "$install_log" 2>&1
```
Comment (Tarun-Arora) [inline install/install.sh:89]: Currently, this doesn't work, I am forced to use a blank password, it says password authentication failed.
I'll try to fix this in this PR only...

Comment (thealphadollar) [reply install/install.sh:89]: @Tarun-Arora Let me know when you have fixed the error.

Comment (Tarun-Arora) [reply install/install.sh:89]: @thealphadollar I have fixed the issue.

**Pull request merged**